### DISCUSSION TITLE
[meta issue] track down & close stalled/obsolete issues to clean up the issue tracker

### DISCUSSION THREAD

**Discussion by ia** [Ralim/IronOS#1751] [General]
**Hello. Just go to TL;DR if you don't want to read my rant** :)

Otherwise... I'm kinda relatively new here. Unfortunately I'm not as good in low-level hardware microelectronics as others. But this project is so awesome that I try to help it in every way possible - with docs, scripts, build system, tiny improves in code where I'm capable.

Recently I had some time for the issue tracker itself. **I looked through every comment in every opened issue** (some of the comments were super-interesting by the way, so many interesting people in the community there), I kid you not! So you wouldn't have to. Because I'm fully aware that the crew is very busy with more important & useful things. However I noticed that there are a lot of opened issues where _there is no any update_, even from the reporters, **even when @discip or @River-Mochi very kindly & very politely asked the reporters to update the info on the report**.

Key take-aways from some issues if I would have been asked to summarize:
- unfortunately, implementing in software/firmware some things which can be done only by hardware - is just impossible (me myself from time to time not fully realize and/or forget that);
- software/firmware can't implement every single _nuance_ of every QC/PD device, hence probably it should be added to somewhere into documentation maybe right in the middle of index.md in bold - if the iron works properly with one power source/cable but not with another one, it's a power source issue unfortunately, but not IronOS one; oh... and that using laptops' ports as high wattage power source for the irons with over ~300 *C temp. just _not a good idea_.

# **TL; DR:**
Here is a _brief_ list of the opened issues which, in my humble opinion, just _should be closed forcefully_ by the project owners who have the permission to do that:

### to close as solved/fixed/implemented:

- https://github.com/Ralim/IronOS/issues/349 - implemented

- https://github.com/Ralim/IronOS/issues/360 - now it's better :)

- https://github.com/Ralim/IronOS/issues/736 - implemented

- https://github.com/Ralim/IronOS/issues/1356 - implemented (this info is in the official documentation now)

- https://github.com/Ralim/IronOS/issues/1510 - implemented (~I have TS80P with SC7A20 and it works bearable~ funny, with latest fw it says LIS2 clone, lol)

- https://github.com/Ralim/IronOS/issues/1552 - implemented

### to close as not relevant/out-of-dated:

- https://github.com/Ralim/IronOS/issues/444

- https://github.com/Ralim/IronOS/issues/518 - not IronOS scope

- https://github.com/Ralim/IronOS/issues/564 - there is the table in the official documentation now

- https://github.com/Ralim/IronOS/issues/583 - impossible to implement considering the complexity of hardware when dealing with QC/PD/etc.

- https://github.com/Ralim/IronOS/issues/619 - not IronOS scope

- https://github.com/Ralim/IronOS/issues/628 - just out-of-date

- https://github.com/Ralim/IronOS/issues/504 - should be closed as a duplicate of 
https://github.com/Ralim/IronOS/issues/227 ;)

- https://github.com/Ralim/IronOS/issues/662 - reported version is obsolete & no response from the reporter :(

- https://github.com/Ralim/IronOS/issues/666 - out-of-dated

- https://github.com/Ralim/IronOS/issues/673 - not relevant

- https://github.com/Ralim/IronOS/issues/698 - it could be a problem with usb cable, usb port, OS drivers for DFU disk... so may not quite relevant to IronOS

- https://github.com/Ralim/IronOS/issues/762 - it's by far not recommended way to power the iron no matter how smart the iron is so not providing compliant QC/PD for TS80P by MBP is a problem of MBP not IronOS

- https://github.com/Ralim/IronOS/issues/810 - discussion, not a bug

- https://github.com/Ralim/IronOS/issues/833 - not relevant

- https://github.com/Ralim/IronOS/issues/839 - probably hardware-related

- https://github.com/Ralim/IronOS/issues/841 - powerbank issue. I have exactly the same powerbank (Tronsmart PBT12, can make a photoproof) - unfortunately it's not _fully_ compatible with declared QC on practice and yes, the same issue with my TS80P regardless powerpulse settings (did try a lot of possible combinations though) on IronOS - PBT12 keeps turning off.

- https://github.com/Ralim/IronOS/issues/907 - there is no software nor hardware which would replace _**safety first**_ approach. :)

- https://github.com/Ralim/IronOS/issues/1032 - no response from the reporter :(

- https://github.com/Ralim/IronOS/issues/1376 - no response from the reporter :(

- https://github.com/Ralim/IronOS/issues/1394 - not IronOS scope: _the firmware_ is not capable to implement in _hardware_ QC3 for MHP30 - MHP30 manufacturer declares PD support only

- https://github.com/Ralim/IronOS/issues/1411 - unclear report, no response :(

- https://github.com/Ralim/IronOS/issues/1419 - looks more like powerbank issue

- https://github.com/Ralim/IronOS/issues/1424 - no response from the reporter :(

- https://github.com/Ralim/IronOS/issues/1484 - more like Pinecil hardware discussion than IronOS issue

- https://github.com/Ralim/IronOS/issues/1508 - looks like another charger problems unfortunately :(

- https://github.com/Ralim/IronOS/issues/1580 - hardware is not compatible for IronOS support

- https://github.com/Ralim/IronOS/issues/1595 - as the author of STM32 Soldering Station Firmware project said: "I can't see any possible cooperation because the targetted devices are completely different".

- https://github.com/Ralim/IronOS/issues/1643 - not an issue & no response from the reporter :(

- https://github.com/Ralim/IronOS/issues/1644 - not IronOS scope, power source issue :(

- https://github.com/Ralim/IronOS/issues/1668 - power source/cable issue :(

- https://github.com/Ralim/IronOS/issues/1162 - power source/cable issue :(

### to verify by extra pair of eyes before closing but very suitable candidates:

- https://github.com/Ralim/IronOS/issues/303 - calibration functions have been implemented and they are ok

- https://github.com/Ralim/IronOS/issues/753 - as far as I understand, it's implemented to a certain degree on hardware where it's technically possible

- https://github.com/Ralim/IronOS/issues/757 - needs update but according the the most recent comment it's tips' issue not IronOS

---

Consider this as the first brief run through opened issues - if/once these will be closed, I will double-sweep the rest of opened ones. The idea is to close _this issue_ once at least the ones from the list above will be closed.

However, if this suggestion (just by a person from the side) is wrong in any way and _it's too much_ (i.e. _telling_ the crew ~what to do~ to _close other issues_) then just let me know and I will close this issue in no time myself and won't bother you with that. But if you agree that some old issues should be cleaned up, then here there the list and you would have to dedicate just ~15-20 minutes of your time to click, scroll through (to check after me whether I'm right or wrong) & close them.

As always, can't wait to hear what do you think.

**Comment by ia** [comment]
And I fully understand that people may be very-very busy with their lives & problems so not having time to update the report is ok but in such case if some issue turned out to be still valid it always can be re-opened easily.

**Comment by ia** [comment]
I see that some issues have been closed - I'm glad that you got time to look through the list. As always I'm happy to help with anything I can.
